### DISCUSSION TITLE
how can i get the YOLO badge?

### DISCUSSION THREAD

**Discussion by sqwapped123** [community/community#36971] [Other Feature Feedback, Questions, & Ideas] [answered]
how can i get the YOLO badge?

**Comment by zaahiral** [comment, marked as answer]
to get the YOLO badge, you need to merge the PR (pull request) directly with the public code repository without verification.

**Reply by AlvarezMar** [reply]
And how can I do that, do the changes in the PR have to be like small or something?

**Comment by AbdulAhad365** [comment]
Yes but in this case i guess no editing is required.

**Reply by icichainz** [reply]
True

**Reply by codepilgrimz** [reply]
cool

**Comment by Khushboqof** [comment]
Merged a pull request without a review

**Comment by AbdulAhad365** [comment]
good
